app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -238,24 +237,13 @@ public void run() {
 
         @Override
         public boolean onLongClick(View v) {
-            String origText = _hlEditor.getText().toString();
             final CommonTextActions commonTextActions = new CommonTextActions(_activity, _hlEditor);
-            final TodoTxtTextActions t_this = TodoTxtTextActions.this;
 
             switch (_action) {
-                case R.string.tmaid_todotxt_add_context: {
-                    SearchOrCustomTextDialogCreator.showSttKeySearchDialog(_activity, origText, t_this::doBasicHighlights, false, t_this::selectLines);
-                    return true;
-                }
-                case R.string.tmaid_todotxt_add_project: {
-                    SearchOrCustomTextDialogCreator.showSttKeySearchDialog(_activity, origText, t_this::doBasicHighlights, true, t_this::selectLines);
-                    return true;
-                }
```

Comment:
Now available through filter dialog

Reply:
functionality is still available this way right? I would opt for keeping the context/project buttons in the textaction toolbar still.